generators/generator-base.js

```diff
@@ -262,29 +270,41 @@ Generator.prototype.getAllInstalledLanguages = function () {
  * get all the languages supported by JHipster
  */
```

Comment:
I could add the old method as well and comment it as deprecated if wanted. Should I do that?